README.md

```diff
@@ -1 +1,26 @@
-# node-repl-lab
\ No newline at end of file
+# Running JavaScript Code in Node REPL
+
+## Objectives
+
+1. Use REPL
+1. Execute code in REPL
+1. Exit REPL
+
+## Introduction
+
+Understanding global and scoping will give you a good foundation for future development.
+
+## Instructions
+
+1. Enter REPL
+1. Write Node code to perform math by creating a sum function
+1. Call the sum function to add numbers
+2. Find out what the day of the week was the day of the Declaration of Independence (July 4th, 1776)
```

Comment:
how do you want students to do this? define a function? how do they even begin to solve that problem? we don't want students to struggle over solving a problem when the real goal is to get them familiar with the node repl.